<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>361</width>
      <height>80</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="m_resource.qrc">:/Iccon/Picresource/Log.jpg</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="Probe_pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>外部测量</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Cali_pushButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>阻值校正</string>
    </property>
   </widget>
   <widget class="QPushButton" name="MainExit_pushButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DataQuery_pushButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>数据查询</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TCRmeasure_pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TCR测量</string>
    </property>
   </widget>
   <widget class="QPushButton" name="TCRsetting_pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>120</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>宋体</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TCR设定</string>
    </property>
   </widget>
   <widget class="QLabel" name="ver_label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>320</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>V1.0.0</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>390</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="m_resource.qrc"/>
 </resources>
 <connections/>
</ui>
